design/ceph/ceph-cluster-cleanup.md

```diff
@@ -38,22 +41,29 @@ spec:
     useAllNodes: true
     useAllDevices: true
   cleanupPolicy:
-    deleteDataDirOnHosts: yes-really-destroy-data
+    confirmation: yes-really-destroy-data
+    sanitizeDisks:
+      method: quick
+      dataSource: zero
+      iteration: 1
+    allowUninstallWithVolumes: false
 ```
 
-- Adding `cleanupPolicy` would cause the operator to refuse running an orchestration
+- Updating the cluster `cleanupPolicy` with `confirmation: yes-really-destroy-data` would cause the operator to refuse running any further orchestration
+- Adding `sanitizeDisks` option to cluster would clean up the disks where the OSDs were configured.
```

Comment:
Even if user don't specify `sanitizeDisk`, cleanup job wipe OSDs, at least in my local environment (k8s 1.18.6 and Rook v1.4.7).

Reply:
Yes, that's correct. I wasn't aware of that. Update the doc to fix it.